goty-client/src/app/form/radio/radio.component.html

```diff
@@ -0,0 +1,10 @@
+<app-info [title]="title">
+  <div class="p-field-radiobutton">
+    <p-radioButton id="yes" name="city" [value]="true" [(ngModel)]="value" inputId="yes"></p-radioButton>
+    <label class="label" for="yes">Yes</label>
```

Comment:
Shouldn't this tie the label to the button? When I click on the label, it doesn't select the radio button.

Reply:
Fixed